### PR TITLE
Added .s4ext file for new extension Breast DCE-MRI FTV

### PR DESCRIPTION
**Pull request by rnadkarni2**
Thank you for contributing to 3D Slicer!

- **To add a new extension with this pull request**: Please keep content of "TODO list for submitting a new extension" section and put an 'x' character in the brackets for each todo item to indicate that you have accomplished that prerequisite.

- **To update an existing extension with this pull request**: Please delete all text in this template and just describe which extension is updated and optionally tell us in a sentence what has been changed. To make extension updates easier in the future (so that you don't have to submit a new pull request after each change in your extension), you may consider replacing specific git hash in your s4ext file by a branch name (for example: `master` or `release` for nightly version; `(majorVersion).(minorVersion)` such as `4.10` for stable Slicer version).

# TODO list for submitting a new extension

[To make sure users can find your extension, understand what it is intended for and how to use it, please complete the checklist below. You do not need to complete all the item by the time you submit the pull request, but most likely the changes will only be merged if all the tasks are done. See more information about the submission process here: https://slicer.readthedocs.io/en/latest/developer_guide/extensions.html]

- [x ] Extension has a reasonable name (not too general, not too narrow, suggests what the extension is for)
- [x ] Repository name is Slicer+ExtensionName
- [x ] Repository is associated with `3d-slicer-extension` GitHub topic so that it is listed [here](https://github.com/topics/3d-slicer-extension). To edit topics, click the settings icon in the right side of "About" section header and enter `3d-slicer-extension` in "Topics" and click "Save changes". To learn more about topics, read https://help.github.com/en/articles/about-topics
- [x ] Extension description summarizes in 1-2 sentences what the extension is usable (should be understandable for non-experts)
- [ ] Any known related patents must be mentioned in the extension description. (not applicable for this extension)
- [ ] If source code license is more restrictive for users than BSD, MIT, Apache, or 3D Slicer license then the name of the used license must be mentioned in the extension description. (not applicable for this extension)
- [x ] Extension URL and revision (scmurl, scmrevision) is correct, consider using a branch name (master, release, ...) instead of a specific git has to avoid re-submitting pull request whenever the extension is updated
- [x ] Extension icon URL is correct (do not use the icon's webpage but the raw data download URL that you get from the download button - it should look something like this: https://github.com/user/project/raw/main/path/to/icon.png)
- [x ] Screenshot URLs (screenshoturls) are correct, contains at least one
- [x ] Homepage URL points to valid webpage containing the following:
  - [x ] Extension name
  - [x ] Short description: 1-2 sentences, which summarizes what the extension is usable for
  - [x ] At least one nice, informative image, that illustrates what the extension can do. It may be a screenshot.
  - [x ] Description of contained modules: at one sentence for each module
  - [x ] Tutorial: step-by-step description of at least the most typical use case, include a few screenshots, provide download links to sample input data set (didn't provide download link, but can use this on DCE-MRI exams from TCIA)
  - [x ] Publication: link to publication and/or to PubMed reference (if available) (added name of paper that will be submitted)
  - [x ] License: We suggest you use a permissive license that includes patent and contribution clauses.  This will help protect developers and ensure the code remains freely available.  We suggest you use the [Slicer License](https://github.com/Slicer/Slicer/blob/master/License.txt) or the [Apache 2.0](https://www.apache.org/licenses/LICENSE-2.0). Always mention in your README file the license you have chosen.  If you choose a different license, explain why to the extension maintainers. Depending on the license we may not be able to host your work. Read [here](https://opensource.guide/legal/#which-open-source-license-is-appropriate-for-my-project) to learn more about licenses.
  - [x ] Content of submitted s4ext file is consistent with the top-level CMakeLists.txt file in the repository (description, URLs, dependencies, etc. are the same)

[If you have any questions or comments then please describe them here.]

### CODE REVIEW

File: Breast_DCEMRI_FTV.s4ext [hunk 1]
```diff
@@ -0,0 +1,44 @@
+#
+# First token of each non-comment line is the keyword and the rest of the line
+# (including spaces) is the value.
+# - the value can be blank
+#
+
+# This is source code manager (i.e. svn)
+scm git
+scmurl https://github.com/rnadkarni2/SlicerBreast_DCEMRI_FTV/tree/master
```
Comment (jamesobutler) [inline Breast_DCEMRI_FTV.s4ext:9]:
```suggestion
scmurl https://github.com/rnadkarni2/SlicerBreast_DCEMRI_FTV
```

File: Breast_DCEMRI_FTV.s4ext [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+# list dependencies
+# - These should be names of other modules that have .s4ext files
+# - The dependencies will be built first
+depends NA
+
+# Inner build directory (default is ".")
+build_subdirectory .
+
+# homepage
+homepage https://github.com/rnadkarni2/SlicerBreast_DCEMRI_FTV/tree/master
```
Comment (jamesobutler) [inline Breast_DCEMRI_FTV.s4ext:21]:
```suggestion
homepage https://github.com/rnadkarni2/SlicerBreast_DCEMRI_FTV
```

File: Breast_DCEMRI_FTV.s4ext [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+# Firstname1 Lastname1 ([SubOrg1, ]Org1), Firstname2 Lastname2 ([SubOrg2, ]Org2)
+# For example: Jane Roe (Superware), John Doe (Lab1, Nowhere), Joe Bloggs (Noware)
+contributors Rohan Nadkarni (UCSF Breast Imaging Research Group), David Newitt (UCSF Breast Imaging Research Group)
+
+# Match category in the xml description of the module (where it shows up in Modules menu)
+category FTV Segmentation
+
+# url to icon (png, size 128x128 pixels)
+iconurl https://github.com/rnadkarni2/SlicerBreast_DCEMRI_FTV/raw/master/Breast_DCEMRI_FTV.png
```
Comment (jamesobutler) [inline Breast_DCEMRI_FTV.s4ext:31]:
```suggestion
iconurl https://raw.githubusercontent.com/rnadkarni2/SlicerBreast_DCEMRI_FTV/master/Breast_DCEMRI_FTV.png
```

File: Breast_DCEMRI_FTV.s4ext [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+# Give people an idea what to expect from this code
+#  - Is it just a test or something you stand behind?
+status 
+
+# One line stating what the module does
+description This extension contains both of the modules needed to compute functional tumor volume (FTV) from a dynamic contrast-enhanced magnetic resonance imaging (DCE-MRI) exam of a breast cancer patient.  The current version of this extension is intended for use on bilateral breast MR exams with axial slices.
+
+# Space separated list of urls
+screenshoturls https://github.com/rnadkarni2/SlicerBreast_DCEMRI_FTV/raw/master/Module1Screenshot.png https://github.com/rnadkarni2/SlicerBreast_DCEMRI_FTV/raw/master/Module2Screenshot.png
```
Comment (jamesobutler) [inline Breast_DCEMRI_FTV.s4ext:41]:
```suggestion
screenshoturls https://raw.githubusercontent.com/rnadkarni2/SlicerBreast_DCEMRI_FTV/master/Module1Screenshot.png https://raw.githubusercontent.com/rnadkarni2/SlicerBreast_DCEMRI_FTV/master/Module2Screenshot.png

```

**Review by pieper — APPROVED** [review]
Other than the couple things @jamesobutler pointed out this looks great to me 👍

**Pull request merged**